packages/grid/_modules_/grid/components/toolbar/GridToolbarDensitySelector.tsx

```diff
@@ -21,7 +21,10 @@ export const GridToolbarDensitySelector = React.forwardRef<HTMLButtonElement, Bu
     const densityValue = useGridSelector(apiRef, gridDensityValueSelector);
     const densityButtonId = useId();
     const densityMenuId = useId();
-    const [anchorEl, setAnchorEl] = React.useState(null);
+
+    const [open, setOpen] = React.useState(false);
+    const buttonRef = React.useRef<HTMLButtonElement>(null);
+    const forkedButtonRef = useForkRef(ref, buttonRef);
```

Comment:
The return value is commonly named `handleRef` in the core.

```suggestion
    const handleRef = useForkRef(ref, buttonRef);
```

https://github.com/mui-org/material-ui/blob/b6977280a603551ff0e5412982b1464b48805cbf/packages/mui-core/src/SliderUnstyled/SliderUnstyled.js#L272